### PR TITLE
Small optimisation in CompilerOptionParseUtilities.cs

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
Just return the array from the `.Split` function as array implements `IList`

### CODE REVIEW

**Comment by AdamSpeight2008** [comment]
Alternatively if we want to return the new list, use `.AddRange` instead.

**Comment by AdamSpeight2008** [comment]
@dotnet-bot please retest windows_release_unit32_prtest

**Comment by AdamSpeight2008** [comment]
@dotnet-bot retest windows_release_unit32_prtest please

File: src/Compilers/Core/Portable/InternalUtilities/CompilerOptionParseUtilities.cs [hunk 1]
```diff
@@ -20,13 +20,7 @@ public static IList<string> ParseFeatureFromMSBuild(string features)
             }
 
             var all = features.Split(new[] { ';', ',', ' ' }, StringSplitOptions.RemoveEmptyEntries);
```
Comment (gafter) [inline src/Compilers/Core/Portable/InternalUtilities/CompilerOptionParseUtilities.cs]: >all [](start = 16, length = 3)

What is the purpose of the variable `all`?

Comment (AdamSpeight2008) [inline src/Compilers/Core/Portable/InternalUtilities/CompilerOptionParseUtilities.cs]: You'll have ask the original implementor, I'm just keeping the existing variable names.
There are other places in the source, where a "temp" is declared just before returning from a function. Think it help when debugging, to have a place where you can attach a breakpoint.

Comment (gafter) [inline src/Compilers/Core/Portable/InternalUtilities/CompilerOptionParseUtilities.cs]: The variable had an obvious purpose in the previous iteration before your change. If it has no purpose now, it should probably be removed (inlined).

---
In reply to: [80747682](https://github.com/dotnet/roslyn/pull/13975#discussion_r80747682) [](ancestors = 80747682)

Comment (gafter) [inline src/Compilers/Core/Portable/InternalUtilities/CompilerOptionParseUtilities.cs]: The variable had an obvious purpose in the previous iteration before your change. If it has no purpose now, it should probably be removed (inlined).

**Comment by gafter** [comment]
LGTM

@dotnet/roslyn-compiler Please review this tiny change submitted by a community member.

**Comment by cston** [comment]
LGTM

**Comment by gafter** [comment]
@dotnet-bot test perf_correctness_prtest please

**Comment by AlekseyTs** [comment]
LGTM

**Pull request merged**